src/uadk_utils.c

```diff
@@ -56,7 +59,9 @@ static void *memcpy_large(void *dstpp, const void *srcpp, size_t len)
 
 void *uadk_memcpy(void *dstpp, const void *srcpp, size_t len)
 {
-	if (len >= UADK_MEM_IMPROVE_THRESHOLD)
+	/* No assembly language is required as glibc version >= 2.32 */
+	if (((__GLIBC__ == GLIBC_MAIN && __GLIBC_MINOR__ < GLIBC_MINOR) ||
```

Comment:
Don't make this a runtime comparison. Make it as a compilation time preprocessing. Refer this:
https://chromium.googlesource.com/v8/v8/+/refs/heads/main/include/v8config.h#36

Reply:
done